mlflow/store/tracking/sqlalchemy_store.py

```diff
@@ -429,6 +429,22 @@ def delete_run(self, run_id):
             run.lifecycle_stage = LifecycleStage.DELETED
             self._save_to_db(objs=run, session=session)
 
+    def _hard_delete_run(self, run_id):
+        """
+        Permanently delete a run (metadata and metrics, tags, parameters).
+        This is used by the mlflow gc command line and is not intented to be used elsewhere.
+        """
+        with self.ManagedSessionMaker() as session:
+            run = self._get_run(run_uuid=run_id, session=session)
+            session.delete(run)
+
+    def _get_deleted_runs(self):
+        with self.ManagedSessionMaker() as session:
+            return session\
+                .query(SqlRun.run_uuid) \
+                .filter(SqlRun.lifecycle_stage == 'deleted') \
```

Comment:
Nit: Can we use `LifecycleStage.DELETED` instead?

Reply:
Done